docker/Dockerfile

```
@@ -1,11 +1,13 @@
 FROM debian:bookworm as builder
 
-RUN apt update && \
-    apt install -y curl gcc python3 libpython3-dev && \
+# hadolint ignore=DL3008
+RUN apt-get update && \
+    apt-get install --no-install-recommends -y curl gcc python3 libpython3-dev && \
```

```suggestion
    apt-get install --no-install-recommends -y ca-certificates curl gcc python3 libpython3-dev && \
```

Without `ca-certificates`, the following error is thrown while building the image:

```
 > [builder  3/11] RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.81.0 && . "${HOME}/.cargo/env":
0.621 curl: (77) error setting certificate file: /etc/ssl/certs/ca-certificates.crt
```

---

Thanks!